### PR TITLE
[PE-4151] Add support for portal styled modals

### PR DESCRIPTION
**Pull request by jllr**
https://ctl.atlassian.net/browse/PE-4151

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/551/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/563/ for more details.

File: src/website/views/components/modal.pug [hunk 1]
```diff
@@ -433,3 +433,39 @@ p.-text Remove <code>a-modal__header</code> and <code>a-modal__footer</code> for
...
+              <div class="a-btn__content">
+                <i class="a-icon icon-x"></i>
+              </div>
+            </button>
+          </header>
+          <div class="a-modal__content">
+            <p class="-text -m--0">Modal content</p>
+          </div>
+          <footer class="a-modal__footer">
+            <button class="a-btn -primary -ml--2">Save</button>
```
Comment (mattnickles) [inline src/website/views/components/modal.pug]: Remove `-ml--2` class. Add `-lg` class to match button size in mockup.

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/578/ for more details.

**Pull request merged**